Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1221,6 +1221,26 @@
     Telecrystal: 2
   categories:
   - UplinkArmor
+  conditions:
+  - !type:StoreWhitelistCondition
+    blacklist:
+      tags:
+      - NukeOpsUplink
+
+- type: listing
+  id: UplinkClothingNoSlipsShoesNukie
+  name: uplink-clothing-no-slips-shoes-name
+  description: uplink-clothing-no-slips-shoes-desc
+  productEntity: ClothingShoesChameleonNoSlips
+  cost:
+    Telecrystal: 4
```

```suggestion
  cost:
    Telecrystal: 2
```

Noslips are fine at 2tc, I don't really see an issue with them.